graph/src/main.py

```diff
@@ -15,22 +15,25 @@
 @app.route('/data')
 @cross_origin()
 def data_index():
-    my_output = json.loads(input_data)
+    content = get_json_content(os.path.join(ui_static_content, "fake_graph.json"))
+    print(content)
+    my_output = content
     validate(
         instance=my_output,
         schema=get_json_content("../contracts/graph-to-ui.json")
     )
     return {'my data json': my_output}
 
-# TODO: serve static content on /
+
 @app.route('/')
 def index():
-    return {"Status": "ok"}
+    return render_template("index.html", path_to_static=ui_static_content)
```

Comment:
nit: I'm not supper familiar with Flask - do we need `render_template`? If that's their way of saying "serve file" I'm cool with that, just asking ;)

Reply:
That's one way to say 'serve file', there is also something like "send_static_file" but it doesn't work xD I can look at it later.